src/main/java/com/tabnine/statusBar/StatusBarEmptySymbolGenerator.java

```diff
@@ -0,0 +1,20 @@
+package com.tabnine.statusBar;
+
+import java.util.Arrays;
+import java.util.List;
+
+// Workaround to address the issue where IJ does not render the width of the status bar icon
+// correctly when the icon changes
+// because it doesn't detect a change in the widget text (it renders it correctly just after
+// document change occurs).
+// Creating a different empty text each time will solve the problem.
+public class StatusBarEmptySymbolGenerator {
```

Comment:
also, please re-implement in kotlin, and this can be a global function without a class.

Reply:
why not a class?

Reply:
why have a class when you dont have to 🙄 
that's my personal preference - you don't have to do it if you think otherwise, I approved you pr anyway 😗